extensions/src/main/java/org/pentaho/platform/plugin/action/WorkItemRequestPayload.java

```diff
@@ -0,0 +1,101 @@
+/*!
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2017 Pentaho Corporation..  All rights reserved.
+ */
+
+package org.pentaho.platform.plugin.action;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.pentaho.platform.api.engine.IPentahoSystemListener;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class WorkItemRequestPayload {
+
+
+  private final String actionId = "actionId";
+  private final String actionUser = "actionUser";
+  private final String actionClass = "actionClass";
+  private final String actionParams = "params";
+  private final String environmentVariablesKey = "environmentVariables";
```

Comment:
These are constants, they should be final. Also, once they're final, they should be ALL_CAPS with underscores for word separation - standard practice.